### PR TITLE
correct class for default file icon

### LINKED ISSUE #90: Default attachment icon not rendering
the default icon is set to `file` in [WPML_Attachment]( https://github.com/No3x/wp-mail-logging/blob/release/1.9.0/src/WPML_Attachment.php#L126) but this results in the class `fa-file-file-o` in the resulting icon element, which does not exist (this should be `fa-file-o`). This renders the attachments invisible.

### PR DESCRIPTION
**Pull request by Spreeuw**
fixes #90

### CODE REVIEW

**Comment by Spreeuw** [comment]
I thought it would make sense to have the icon class method return the actual _class_ (separating from the method to determing the mime _type_), but feel free to divert :)

File: src/WPML_Attachment.php [hunk 1]
```diff
@@ -62,7 +62,8 @@ public function getUrl() {
      */
     public function getIconClass() {
         if ($this->iconClass === null) {
-            $this->iconClass = $this->determine_mime_icon_class($this->getPath());
+            $mimeIconType = $this->determine_mime_icon_type($this->getPath());
```
Comment (No3x) [inline src/WPML_Attachment.php]: My intention was to keep the icon class generic (not fa-specific) in the Attachment class and only use the fa information in the rendering process (https://github.com/No3x/wp-mail-logging/blob/4bffb8b/src/WPML_Utils.php#L58). I would like to see this distinction in the WPML_Utils class.

File: src/WPML_Utils.php [hunk 2]
```diff
@@ -55,7 +55,7 @@ public static function recursive_array_search( $needle, $haystack ) {
      * @return string returns the most suitable fa icon or generic one if not possible.
      */
     public static function determine_fa_icon( $iconClass ) {
-        return '<i class="fa fa-file-' . esc_attr($iconClass) . '-o"></i>';
```
Comment (No3x) [inline src/WPML_Utils.php]: A test for this would be great to be sure this doesn't happen again.

File: src/WPML_Attachment.php (hunk 1 above)
Comment (Spreeuw) [reply src/WPML_Attachment.php]: fair enough & agreed - like I said, feel free to diver

**Pull request closed without merging**